azure-resources/KeyVault/vaults/recommendations.yaml

```diff
@@ -1,21 +1,21 @@
-- description: Key vaults should have soft delete enabled
-  aprlGuid: 1cca00d2-d9ab-8e42-a788-5d40f49405cb
+- description: Azure Key Vault RBAC for Data Plane
```

Comment:
Could you please remove the soft-delete kql file and then add a new kql file with the new recommendation guide.

Reply:
It doesn't look like the KQL file was removed for soft delete. There also wasnt a new query file added for the new data plane recommendation.

Also, could you please reach out to me in Teams (ztrocinski) so I can confirm your Microsoft alias.